paddle/operators/to_array_op.cc

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/operators/to_array_op.h"
+#include "paddle/framework/data_type.h"
+
+namespace paddle {
+namespace operators {
+
+class ToArrayOp : public framework::OperatorWithKernel {
```

Comment:
This op should be an `Operator` rather than an `OperatorWithKernel`. Because only `MemCopy` has been used.